numba_dpex/tests/kernel_tests/test_arg_types.py

```diff
@@ -53,6 +55,9 @@ def check_bool_kernel(A, test):
         A[0] = 222
 
 
+@pytest.mark.skipif(
```

Comment:
We have to investigate it further, in the past we have had code generation issues caused due to improper pointer width for 64bit ints getting generated on Window. 

As a first step, can you convert the test to use dpnp/dpctl.tensor and compute follows data. If the issue is not showing up with dpnp/dpctl.tensor we are golden.